packages/zoe/src/zoeService/zoeSeat.js

```diff
@@ -78,6 +78,7 @@ export const makeZoeSeatAdminKit = (
       updater.fail(reason);
       doExit(zoeSeatAdmin);
     },
+    getNotifier: async () => notifier,
```

Comment:
```suggestion
    getNotifier: () => Promise.resolve(notifier),
```

Reply:
I'm sure @erights can explain it better but I'll take a shot…

Better, returning synchronously.  Check out https://es6console.com/l5qxpqfh/ and "transform".

This ES2017:
```
const val = 'hi';

const asyncDemo = async () => val;
const promiseDemo = Promise.resolve(val);
```

turns into this ES5:
```
function _asyncToGenerator(fn) { return function () { var gen = fn.apply(this, arguments); return new Promise(function (resolve, reject) { function step(key, arg) { try { var info = gen[key](arg); var value = info.value; } catch (error) { reject(error); return; } if (info.done) { resolve(value); } else { return Promise.resolve(value).then(function (value) { return step("next", value); }, function (err) { return step("throw", err); }); } } return step("next"); }); }; }

const val = 'hi';

const asyncDemo = (() => {
  var ref = _asyncToGenerator(function* () {
    return val;
  });

  return function asyncDemo() {
    return ref.apply(this, arguments);
  };
})();
const promiseDemo = Promise.resolve(val);
```